packages/ember-glimmer/lib/environment.js

```diff
@@ -133,6 +137,18 @@ export default class Environment extends GlimmerEnvironment {
           wrapClassAttribute(args);
           return new CurlyComponentSyntax({ args, definition, templates });
         }
+      } else {
+        // Check if it's a keyword
+        let mappedKey = keywords[key];
+        if (mappedKey) {
+          if (mappedKey !== key) {
+            path = path.map((segment) => segment === key ? mappedKey : segment);
```

Comment:
How should this be more generic? Should be able to just add the `input` mapping to the [`keyword` POJO](https://github.com/emberjs/ember.js/pull/13488/files#diff-c8938eb5cb17b733982744e1621cb8e4R34) to get input working.